schema/artist/index.js

```diff
@@ -103,11 +111,38 @@ const ArtistType = new GraphQLObjectType({
       deathday: {
         type: GraphQLString,
       },
+      biography: {
+        type: Article.type,
+        description: 'The Artist biography article written by Artsy',
+        resolve: ({ _id }) => {
+          return positron('articles', {
+            published: true,
+            biography_for_artist_id: _id,
+            limit: 1,
+          }).then(articles => first(articles.results));
+        },
+      },
       alternate_names: {
         type: new GraphQLList(GraphQLString),
       },
       meta: Meta,
-      blurb: markdown(),
+      blurb: {
```

Comment:
makes sense, if there's a blurb use it, otherwise return the biography for the selected partner artist.

The blurb on an artist can be markdown, but the bio on partner artist is just plain text?